data/sql/headers.go

```diff
@@ -182,6 +183,7 @@ func NewHeadersDb(db *sqlx.DB, dbType vconfig.DbType) *HeadersDb {
 				insertBH: sqliteInsertHeader,
 			},
 		},
+		logger: lf.NewLogger("headers-db"),
```

Comment:
can you name it simply `log`?
```suggestion
		log: lf.NewLogger("headers-db"),
```

Reply:
Done